app/views/organisation_member/requests/show.html.erb

```diff
@@ -18,30 +18,29 @@
     <div class="row">
       <div class="col-sm-6">
         <dt>Requester</dt>
-        <dd>
+        <dd class="border-bottom">
           <%= @request.requester.name %>
         </dd>
         <dt>Requestee</dt>
-        <dd>
+        <dd class="border-bottom">
           <%= @request.requestee.email %>
         </dd>
         <dt>Note</dt>
-        <dd>
+        <dd class="border-bottom">
           <%= @request.note %>
         </dd>
         <dt>State</dt>
-        <dd>
+        <dd class="border-bottom">
           <%= @request.state %>
         </dd>
         <dt>Document type</dt>
-        <dd>
+        <dd class="border-bottom">
```

Comment:
how about putting border-bottom on the <dd> in the css.
This will be in one place (the css/scss file) instead of on every tag.

Reply:
:+1: